src/actions/Entities/NameActions.js

```diff
@@ -11,9 +11,10 @@ export const NAME_ACTIONS = {
   FILTER: 'NAME/filter',
   SORT: 'NAME/sort',
   SAVE: 'NAME/save',
+  FETCH_SUCCESS: 'NAME/fetchSuccess',
 };
 
-const createDefaultName = (type = 'patient', id) => ({
+export const createDefaultName = (type = 'patient', id) => ({
```

Comment:
rather than passing an id in and creating one here, started to create a default one in components and pass the patient.

This is because I would be passing a patient when selecting one from having been lookedup.

Just occurs to me I mightv'e been able to just quickly save the patient when selecting and made a lot less changes 🤔